src/Pages/TemplatesTable/components/AddTemplate/steps/SetUpDateStep.tsx

```diff
@@ -83,7 +83,10 @@ export default function SetUpDateStep() {
 
   const columnHeaders = ['Name', /* 'Label',*/ 'Advisories', 'Packages'];
 
-  const itemsAfterDate = useMemo(() => data?.filter(({ is_after }) => is_after) || [], [data]);
+  const itemsAfterDate = useMemo(() => data?.data?.filter(({ is_after }) => is_after) || [], [data?.data]);
+  console.log("AFTER")
```

Comment:
i'd remove these console.log statements once you're finished testing

Reply:
![giphy](https://github.com/content-services/content-sources-frontend/assets/395077/ced541e4-ff6f-4bcb-9479-e089e8fa48ed)

Thanks!